### PR TITLE
Vas 35/feat/patient record

### PR DESCRIPTION
**Pull request by Panas18**

### CODE REVIEW

**Review by aayush-musyaju — APPROVED** [review]
Everything looks fine to me !!

**Review by crazieeKinG — APPROVED** [review]
Everything looks fine

**Review by ManojMoktan1 — APPROVED** [review]
Looks good!

File: src/components/navbar/navbar.tsx [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
 export const Navbar: React.FC = () => {
   const dispatch = useDispatch();
-  const isLoggedIn = useSelector((state: RootState) => state.isLoggedIn.isAuth);
+  // const isLoggedIn = useSelector((state: RootState) => state.isLoggedIn.isAuth);
   const navigation = useNavigate();
+
   const handleLogout = () => {
     dispatch(logoutAction());
     localStorage.setItem("isLoggedIn", JSON.stringify(false));
     navigation({ pathname: "/login" });
```
Comment (RikLakhe) [inline src/components/navbar/navbar.tsx:17]: why done be have the logic of cleaning up the localstory inside logoutaction ?

Comment (Panas18) [reply src/components/navbar/navbar.tsx:17]: Is it required to clean local storage inside the logout action? I have changed the local storage variable to false inside logout action.

**Pull request merged**